OmiseSDK/Compatibility/OmiseCapability.swift

```diff
@@ -89,6 +89,8 @@ class __OmiseCapabilitySourceBackendPayment: __OmiseCapabilityBackendPayment {
 
     static let FPXSourceBackendPayment =
         __OmiseCapabilitySourceBackendPayment(sourceType: OMSSourceTypeValue.fpx)
+    
+    static let rabbitLinepaySourceBackendPayment = __OmiseCapabilitySourceBackendPayment(sourceType: OMSSourceTypeValue.rabbitLinepay)
```

Comment:
I think we should indent it like the other payment methods above.

Reply:
Fixed -> [cf56173](https://github.com/omise/omise-ios/pull/160/commits/cf5617351b7f43d66d31ac5be5e78cd72fb180aa)